modules/etl/talend/talend-definition/src/main/java/com/liferay/talend/runtime/LiferaySourceOrSinkRuntime.java

```diff
@@ -32,16 +32,30 @@
  */
 public interface LiferaySourceOrSinkRuntime extends SourceOrSink {
 
+	public String getActualWebSiteName(String webSiteURL) throws IOException;
+
 	public List<NamedThing> getAvailableWebSites() throws IOException;
 
 	public Schema getExpectedFormSchema(Operation operation) throws IOException;
 
+	/**
+	 * @deprecated In favor of determining the schema based on the resource type
```

Comment:
@ZoltanTakacs, why are we deprecating this, if these modules have never been published? Shouldn't we just remove it?